src/components/SelectCountry/CountryDropDown.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from "react";
+
+// CountryDropDownProps 타입 지정
+interface ICountryDropDownProps {
+  countries: string[];
+  showDropDown: boolean;
+  toggleDropDown: Function;
+  countrySelection: Function;
+}
+
+const CountryDropDown = ({
+  countries,
+  countrySelection,
+}: ICountryDropDownProps) => {
+  const [showDropDown, setShowDropDown] = useState<boolean>(false);
+
+  const onClickHandler = (country: string) => {
+    countrySelection(country);
+  };
+
+  useEffect(() => {
+    setShowDropDown(showDropDown);
+  }, [showDropDown]);
+
+  return (
+    <div className={showDropDown ? "dropdown" : "dropdown active"}>
+      {countries.map((country, index) => {
+        return (
+          <p
+            key={index}
+            onClick={(): void => {
+              onClickHandler(country);
+            }}
```

Comment:
```suggestion
            onClick={() => {
              onClickHandler(country);
            }}
```

다른 곳들도 이런 게 있는데 void를 쓰시는 이유가 뭔가요?